project.clj

```diff
@@ -5,13 +5,13 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [manifold "0.1.4"]
+                 [manifold "0.1.6"]
                  [org.clojure/math.numeric-tower "0.0.4"]]
-  :plugins [[lein-auto "0.1.2"]
-            [lein-cljfmt "0.3.0"]
-            [lein-pprint "1.1.1"]
-            [lein-environ "1.0.2"]
+  :plugins [[lein-auto "0.1.3"]
+            [lein-cljfmt "0.5.6"]
```

Comment:
cljfmt 0.5.6 has an incompatibility with lein-ancient iirc which is causing Travis to explode